### PR TITLE
enable dependabot scheduled updates

### LINKED ISSUE #164: Add dependabot to automatically update dependencies
Explore how to setup dependabot for automatic updates of
* client-go
* controller-runtime

### PR DESCRIPTION
**Pull request by jbpratt**
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file

closes #164

Signed-off-by: jbpratt <jbpratt78@gmail.com>

### CODE REVIEW

**Comment by vladimirvivien** [comment]
@cpanato please take a look at GitHub YAML.

**Comment by vladimirvivien** [comment]
@cpanato or @ShwethaKumbla PTAL

**Review by cpanato — APPROVED** [review]
/lgtm

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
+    directory: /
+    schedule:
+      interval: daily
```
Comment (cpanato) [inline .github/dependabot.yml:6]: sorry, a minor change, can you add the following

```
  labels:
    - "area/dependency"
    - "release-note-none"
    - "ok-to-test"
```

/lgtm cancel

Comment (jbpratt) [reply .github/dependabot.yml:6]: added the labels!

Comment (cpanato) [reply .github/dependabot.yml:6]: thanks!

**Review by cpanato — APPROVED** [review]
/lgtm
/assign @vladimirvivien

**Comment by vladimirvivien** [comment]
Thanks @cpanato 

/approve

**Pull request merged**